backend/api/views.py

```diff
@@ -107,32 +116,105 @@ def create_job(self, request):
             raise exceptions.NotFound("Selected printer does not exist")
         job = self._create_printing_job(printer_with_perms=printer_with_perms, **serializer.validated_data)
         return Response(self.get_serializer(job).data)
+    
+    @action(detail=True, methods=['post'], name='Change job properties')
+    def change_properties(self, request, pk=None):
+        job = self.get_object()
+        serializer = CreatePrintJobRequestSerializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        self._validate_properties(serializer.validated_data['printer'], PrintingProperties(
+            color=serializer.validated_data['color'],
+            copies=serializer.validated_data['copies'],
+            two_sides=serializer.validated_data['two_sides'],
+            pages_to_print=serializer.validated_data['pages_to_print'],
+            fit_to_page=serializer.validated_data['fit_to_page'],
+        ))
```

Comment:
Maybe this could work a bit differently, for example calling `_validate_properties` inside `_change_properties` before calling `.save()`. The current way of doing it makes it easy to introduce problems in the future if these lines are not synced with `_change_properties`.